blob/fileblob/fileblob.go

```diff
@@ -403,7 +403,15 @@ func (b *bucket) ListPaged(ctx context.Context, opts *driver.ListOptions) (*driv
 func (b *bucket) As(i interface{}) bool { return false }
 
 // As implements driver.ErrorAs.
-func (b *bucket) ErrorAs(err error, i interface{}) bool { return false }
+func (b *bucket) ErrorAs(err error, i interface{}) bool {
```

Comment:
Please also update the package godoc for As.

Reply:
Done.